client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -30,7 +30,13 @@ const AuthHeader = ({ linkTo, btnText }: Props): JSX.Element => {
           minWidth="55vw"
           flexWrap="wrap"
           className={classes.linkContainer}
-        >
+        > 
+          <Link to={{ pathname: '/artist', state:'60f101c8d872991344be3628' }}>
```

Comment:
Lose the hard coded id here.

Reply:
Shit, did i forget that. I thought i removed it :)., I am removing in the next commit. Thank you @tsbolty